Podfile

```diff
@@ -20,7 +20,7 @@ target 'WooCommerce' do
 
   # Use the latest bugfix for coretelephony
   #pod 'Automattic-Tracks-iOS', :git => 'https://github.com/Automattic/Automattic-Tracks-iOS.git', :tag => '0.2.4-beta.1'
-  pod 'Automattic-Tracks-iOS', '~> 0.4.0'
+  pod 'Automattic-Tracks-iOS', '~> 0.4.4-beta'
```

Comment:
I also compared the changes from [0.4.0 to 0.4.4-beta](https://github.com/Automattic/Automattic-Tracks-iOS/compare/0.4.0...0.4.4-beta.2). I didn't see anything to be concerned about.